pkgs/by-name/fe/fertilizer/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "fertilizer";
+  version = "0.3.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "moleculekayak";
+    repo = "fertilizer";
+    tag = "v${version}";
+    hash = "sha256-sDoAjEiKxHf+HtFLZr6RwuXN+rl0ZQnFUoQ09QiE6Xc=";
+  };
+
+  build-system = with python3.pkgs; [ setuptools ];
+
+  dependencies = with python3.pkgs; [
+    bencoder
+    colorama
+    flask
+    requests
+  ];
+
+  nativeCheckInputs = with python3.pkgs; [
+    pytestCheckHook
+    requests-mock
+  ];
+
+  pythonImportsCheck = [ "fertilizer" ];
+
+  meta = {
+    description = "Cross-seeding tool for music";
+    homepage = "https://github.com/moleculekayak/fertilizer";
```

```suggestion
    homepage = "https://github.com/moleculekayak/fertilizer";
    changelog = "https://github.com/moleculekayak/fertilizer/releases/tag/${src.tag}";
```

Could you add changelog to metadata?

---

We must not rely on `src`'s attribute to set `changelog`.

---

I see, I didn't know the problem the override causes. Maybe `let tag = ... in ~` and `inherit tag;` syntax helps this.
The `changelog` field itself is useful, so could you add it to the derivation?